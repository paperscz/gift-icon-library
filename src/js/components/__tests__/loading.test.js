import React from 'react';
import { mount } from 'enzyme';

import Loading from '../loading';

describe('The loading component', () => {
  let component;
  it('should render', () => {
    component = mount(<Loading />);
    expect(component).toMatchSnapshot();
  });
});
