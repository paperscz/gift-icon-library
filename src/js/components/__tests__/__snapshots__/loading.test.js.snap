// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The loading component should render 1`] = `
<Loading>
  <div
    className="loading"
  >
    <h3>
      Application is Loading
    </h3>
  </div>
</Loading>
`;
